tests/e2e/specs/shopper/cart-proceed-to-checkout.test.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { shopper } from '../../../utils';
+import { SIMPLE_PRODUCT_NAME } from '../../../utils/constants';
+
+const block = {
+	name: 'Cart',
+};
+
+if ( process.env.WOOCOMMERCE_BLOCKS_PHASE < 2 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( `skipping ${ block.name } tests`, () => {} );
+
+describe( 'Shopper → Cart → Can proceed to checkout', () => {
+	beforeAll( async () => {
```

Comment:
instead emptying the cart `beforeAll` and `afterAll`, what do you think about emptying the cart in `beforeEach` so that if in the future more tests are added to this file, we don't need to change this?

Reply:
I agree with the `beforeEach`. I'll apply this in my next PR as well! 🙌